app/client/views/settings/data/data.js

```diff
@@ -1,70 +1,138 @@
-import FileSaver from 'file-saver';
+import FileSaver from "file-saver";
 
-Template.dataSettings.onCreated(function () {
+const showAlert = () => {
```

Comment:
This function is probably unnecessary, since the user can dismiss the alert.

In general, less code is better.